### DISCUSSION TITLE
wire:navigate not working with lazy load

### DISCUSSION THREAD

**Discussion by SmoKE585** [livewire/livewire#6758] [7. Bugs (archived)] [answered]
### Livewire version

v3-main

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. index page code
`
@forelse($popularGoods as $good)
    <livewire:components.goods.product-card :good="$good" :key="$good->good_id" lazy/>
@empty

@endforelse
`
2. In product card

> <\a href="{{route('catalog.product', ['good_id' => $good->good_id, 'slug' => \Illuminate\Support\Str::slug($good->name)])}}" wire:navigate>title</a>

In this case, the page is completely updated, wire:navigate dont effect

### Additional information

https://acgame.ru/ home page block with products at the bottom

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joydeep-bhowmik** [comment]
Yeah same here

**Comment by onemoreahmad** [comment]
Same here.

**Comment by joshhanley** [comment, marked as answer]
We're in the process of investigating all wire:navigate issues and I have tested out your scenario. I believe this has been fixed so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
